### PR TITLE
Figures

### PR DESCRIPTION
**Pull request by scseeman**

### CODE REVIEW

File: aisynphys/connectivity.py [hunk 1]
```diff
@@ -160,12 +164,22 @@ def measure_connectivity(pair_groups, alpha=0.05, sigma=None):
...
+            results[(pre_class, post_class)]['probed_distances'] = distances[mask]
+            results[(pre_class, post_class)]['connected_distances'] = connections[mask]
+        if sigma is not None:
             adj_conn_prob, adj_lower_ci, adj_upper_ci = distance_adjusted_connectivity(distances[mask], connections[mask], sigma=sigma, alpha=alpha)
             results[(pre_class, post_class)]['adjusted_connectivity'] = (adj_conn_prob, adj_lower_ci, adj_upper_ci)
+        if fit_model is not None:
+            if sigma is not None:
+                fit = fit_model.fit(distances[mask], connections[mask], method='L-BFGS-B', fixed_size=sigma)
+            else:
+                fit = fit_model.fit(distances[mask], connections[mask], method='L-BFGS-B')
```
Comment (campagnola) [inline aisynphys/connectivity.py:180]: I think you can replace this with just:

    fit = fit_model.fit(distances[mask], connections[mask], method='L-BFGS-B', fixed_size=sigma)

File: aisynphys/connectivity.py [hunk 2]
```diff
@@ -521,4 +535,54 @@ def err_fn(cls, params, *args):
...
+def recip_connectivity_profile(cp_results_1, cp_results_2, bin_edges):
+    """
+    Given connection probability results of two pair groups what is the normalized probability of finding a reciprocal cnx 
+    
+    Parameters
+    ----------
+    cp_results_1 : dict
+        output of measure_connectivity for first connection type (A->B)
+    cp_results_2 : dict
+        output of measure connectivity for second connection type (B->A)
```
Comment (campagnola) [inline aisynphys/connectivity.py:547]: I don't really get why you want to run measure_connectivity on a-b, b-a only to access the lists of pairs and distances? I think it would make sense just to pass in the list of pairs instead.

File: aisynphys/ui/notebook.py [hunk 3]
```diff
@@ -506,36 +506,25 @@ def show_distance_profiles(ax, results, colors, class_labels):
...
-    if true_model is not None:
-        ax.plot(x_vals, true_model.connection_probability(x_vals), color=(0, 0.5, 0))
-
-    # plot the connectivity profile with confidence intervals (black line / grey area)
+   
     _, cprop, lower, upper = connectivity_profile(conn, x_probed, x_bins)
-    ax.plot(x_bins, np.append(cprop, cprop[-1]), drawstyle='steps-post', color=(.5, .5, .5))
-    ax.fill_between(x_bins, np.append(lower, lower[-1]), np.append(upper, upper[-1]), step='post', color=(0.85, 0.85, 0.85))
+    # plot the connectivity profile with confidence intervals (black line / grey area)
+    show_distance_binned_cp(x_bins, cprop, ax, ci_lower=lower, ci_upper=upper)
```
Comment (campagnola) [inline aisynphys/ui/notebook.py:519]: Did you make this change, or did I?

File: aisynphys/connectivity.py (hunk 2 above)
Comment (scseeman) [reply aisynphys/connectivity.py:547]: I suppose I was trying to make it work with our other functions but could do what you suggest.

Comment (scseeman) [reply aisynphys/connectivity.py:547]: So I guess the other reason was because I use other keys from the results dict to plug into `connectivity_profile` so that we're not constantly re-creating those arrays for various other analyses

File: aisynphys/ui/notebook.py (hunk 3 above)
Comment (scseeman) [reply aisynphys/ui/notebook.py:519]: I did...I broke up the 3 components of `show_connectivity_profile`, the fit, the binned Cp, and the raster into 3 separate functions so that I could use them piecemeal elsewhere. You can still use `show_connectivity_profile` to get exactly what it did before.
